<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tron_menu</class>
 <widget class="QMainWindow" name="tron_menu">
  <property name="windowTitle">
   <string>tron by 7324</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="central_layout">
    <!-- Top spacer to center the whole content -->
    <item> <!-- 0 -->
     <spacer name="top_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width> 
       </size>
      </property>
     </spacer>
    </item>

    <!-- Logo label -->
    <item> <!-- 1 -->
     <widget class="QLabel" name="logo_label">
      <property name="text">
       <string>TRON</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <!-- <string notr="true">font-size: 36px; font-weight: bold; color: cyan;</string> -->
      </property>
     </widget>
    </item>

    <!-- Spacer between logo and buttons -->
    <item> <!-- 2 -->
     <spacer name="between_logo_and_buttons">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
       </size>
      </property>
     </spacer>
    </item>

    <!-- Buttons area -->
    <item> <!-- 3 -->
     <layout class="QVBoxLayout" name="main_screen_layout">
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>

      <item> <!-- 4 -->
       <widget class="QPushButton" name="start_button">
        <property name="text">
         <string>start</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>

      <item> <!-- 5 -->
       <widget class="QPushButton" name="settings_button">
        <property name="text">
         <string>settings</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>

      <item> <!-- 6 -->
       <widget class="QPushButton" name="creators_info_button">
        <property name="text">
         <string>creators</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>

      <item> <!-- 7 -->
       <widget class="QPushButton" name="quit_button">
        <property name="text">
         <string>quit</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Bottom spacer to balance the layout -->
    <item> <!-- 8 -->
     <spacer name="bottom_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
